website/project/views/contributor.py

```diff
@@ -505,6 +507,29 @@ def send_claim_email(email, user, node, notify=True, throttle=24 * 3600):
     return to_addr
 
 
+@contributor_added.connect
+def notify_contributor(node, contributor, throttle=24 * 3600):
+    if contributor.is_registered and not node.template_node and not node.is_fork:
```

Comment:
Can you add a brief comment explaining why we exempt templated and forked projects?